### PR TITLE
add cask file for Ultimate app

### PR DESCRIPTION
**Pull request by daehyunpy**
add cask file: ultimate.rb
Ultimate app of Epubor: http://www.epubor.com

### CODE REVIEW

File: Casks/ultimate.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'ultimate' do
+  version '3.0.4.41'
+  sha256 :no_check
+
+  url 'http://download.epubor.com/epubor_ultimate.zip'
+  name 'Ultimate Converter'
+  homepage 'http://www.epubor.com'
+  license :commercial
+
+  container :nested => "epubor_ultimate/Ultimate_v#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/ultimate.rb]: Needs blank line below.

File: Casks/ultimate.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'ultimate' do
+  version '3.0.4.41'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/ultimate.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by vitorgalvao** [comment]
Thank you. Could you [squash your commits](http://davidwalsh.name/squash-commits-git)? Please @mention me after you do, so I get an email to check back.

**Comment by hyuna917** [comment]
@vitorgalvao hmm, i did it. but i’m not sure that i did it correctly.

**Comment by vitorgalvao** [comment]
You added unrelated commits. Don’t worry, and feel free to try again or start over, if you’d like. Rebasing is an obtuse process until you get the hang of it, and most people get confused on the first tries.

**Comment by hyuna917** [comment]
thank you for saying that. i was worried about that i made a mess.

**Comment by vitorgalvao** [comment]
Nah, even so it’s easy to just delete the branch and make a new correct one. Most people go that route when failing a rebase.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 594ad25484a17fc481abba4a2054304877f11f59 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**